apps/picsa-tools/crop-probability-tool/src/app/components/crop-probability-table/crop-probability-table.component.html

```diff
@@ -1,43 +1,85 @@
-<button
-  mat-icon-button
-  color="accent"
-  aria-label="Example icon button with a menu icon"
->
-  <mat-icon>menu</mat-icon>
-</button>
-<table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
-  <!--- Note that these columns can be defined in any order.
-        The actual rendered columns are set as a property on the row definition" -->
+<!-- <div ngClass="table-filter">
+  <mat-form-field>
+    <mat-label>Filter</mat-label>
+    <input
+      matInput
```

Comment:
I can see this is the implementation you had issues with. Taking a quick look your code is good, I think there's just a minor issue because the `matInput` you've referenced hasn't been imported into the material module 

(super annoying I know, so many separate imports and the error message it not even remotely useful - I only realised by chance)

I'll add a commit to include